.eslintrc.json

```diff
@@ -8,9 +8,6 @@
         "plugin:react/recommended",
         "eslint:recommended"
     ],
-    "overrides": [{
-        "files": ["*.jsx", "*.js"]
```

Comment:
These need to get put back into the new overrides located down there VVV

Reply:
👍🏽